docs/orm-integration.md

```diff
@@ -156,7 +270,7 @@ In the generic buyer skill config (`my_buyer_aea/skills/generic_buyer/skill.yaml
 |        search_value: UK           |        search_value: UK          |
 |        constraint_type: '=='      |        constraint_type: '=='     |
 |ledgers: ['fetchai']               |ledgers: ['ethereum']             |
-|----------------------------------------------------------------------| 
+|----------------------------------------------------------------------|
```

Comment:
the above format is a bit un-handy unless someone uses vertical selection.

Reply:
Yes I also don't like it